### DISCUSSION TITLE
Can i get rid of theses badges?

### DISCUSSION THREAD

**Discussion by 3a1b2c3** [community/community#28255] [Other Feature Feedback, Questions, & Ideas]
Thanks

**Comment by airtower-luna** [comment]
You can disable showing them on your profile: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/showing-your-private-contributions-and-achievements-on-your-profile#changing-the-visibility-of-achievements

**Comment by lechithinh** [comment]
Go to https://github.com/settings/profile then scroll down. It gives you an option to turn achievements off.

**Comment by NunoCardoso** [comment]
Can we just please not gamify everything? We are just coding. We do not need to have all our dairy activities sprinkled with carrot-and-stick rewarding gimmicks.

**Comment by jfullstackdev** [comment]
GitHub considers all users, we have different opinions for gamification but others really love it, so it's meeting halfway, that's why they enabled to hide it, you can even make your account private

another one valid request is the ability to select only badges to be displayed because as of now, it's all or nothing, 

another one, even if the account is already private, those who follow this user, just like an inner circle, will still be able to see the activities of that user
